server/markdown.js

```diff
@@ -0,0 +1,78 @@
+"use strict";
+
+var markdownIt  = require("markdown-it")
+var logger      = require("./logger")
+var highlight   = require("highlight.js")
+
+var mdiAnchor   = require("markdown-it-anchor");
+var mdiAttrs    = require("markdown-it-attrs");
+
+const LANGS = {
+	"js": "JavaScript",
+	"ts": "TypeScript",
+	"bash": "Bash",
+	"json": "JSON"
+}
+
+function formatCode(code, lang, filename){
+	return `<dttw-code>
+		<header>
+			<span class="lang">${escape((lang in LANGS ? LANGS[lang] : lang), true)}</span>
+			<span class="copy">
+				<dttw-icon class="material-icons">content_copy</dttw-icon>
+			</span>
+			<span class="filename">${filename || ""}</span>
+		</header>
+		<pre class="hljs"><code class=${escape((lang in LANGS ? LANGS[lang].toLowerCase() : lang), true)}>${code}</code></pre>
+	</dttw-code>`;
+}
```

Comment:
aww man, are we letting es6 into this project now? To my knowledge, we had avoided it thus far. It might make sense to use the old style until node6 has more support